Warn for CRS inaccuracies (#6831)
qgis/QGIS-Documentation: docs/user_manual/introduction/qgis_configuration.rst

* |radioButtonOff| :guilabel:`Use project CRS`
* |radioButtonOff| :guilabel:`Use a default CRS`

.. _crs_inaccuracies:

**Accuracy warnings**

.. A small intro either on accuracy differences between CRS/datum,
   or static vs dynamic CRS would be nice here.
   Or if you want, you can expand a lot in the working_with_projection.rst file.

>    Also if anyone knows a link to "datum ensemble" concept?

Reviewer: Unfortunately not. There's various blog posts which are decent summaries, but nothing formal enough to include in the documentation. (I'd love someone to petition osgeo to fund a well written, peer reviewed intro-level explanation to these concepts)

:guilabel:`Only show warnings for CRS inaccuracies which exceed` a given

> distance: occurs when you are explicitly creating or modifying a data and

Reviewer: ```suggestion
distance: occurs when you are explicitly creating or modifying a dataset and
```

> select in the CRS selector widget a CRS based on a datum ensemble with lower

Reviewer: ```suggestion
select a CRS based on a datum ensemble with lower
```

accuracy. The default is to ``Always show`` the warning if any inaccuracy.
Requires a QGIS version using at least `PROJ 8.0 <https://proj.org/index.html>`_.

|unchecked| :guilabel:`Show warning for CRS inaccuracies for layers in project legend`:
If checked, any layer with a CRS with accuracy issues (i.e. a dynamic crs with

> no coordinate epoch available, or a crs based on a datum ensemble with accuracy

Reviewer: ```suggestion
no coordinate epoch available, or a CRS based on a datum ensemble with inherent inaccuracy
```

exceeding the user-set limit) will have the |indicatorLowAccuracy| warning icon
in the :guilabel:`Layers` panel reflecting that it is a low-accuracy layer.

> This is designed for use in engineering, BIM,... industries where inaccuracies

Reviewer: ```suggestion
This is designed for use in engineering, BIM, asset management, and other fields where inaccuracies
```

> of meter/submeter level are very dangerous.

Reviewer: ```suggestion
of meter/submeter level are potentially very dangerous or expensive!
```
